include/openenclave/internal/syscall/declarations.h

```diff
@@ -209,6 +210,11 @@ OE_DECLARE_SYSCALL2(SYS_link);
 OE_DECLARE_SYSCALL5(SYS_linkat);
 OE_DECLARE_SYSCALL2_M(SYS_listen);
 OE_DECLARE_SYSCALL3(SYS_lseek);
+/*
+ * lstat is required to compile musl/src/stat/fstatat.c, and hence
```

Comment:
Is fstatat.c a hard requirement (i.e., does existing files depend on it)? It's still unclear to me why we need to add the source without implementing the dependent syscalls.

Reply:
The stat.c file uses one function that is present in the fstatat.c file. So we need to be able to compile the src fstatat.c. One of the other functions in fstatat.c that we do not use, need lstat to compile. Hence, we add a shell lstat, but actually don't implement it.